cves/kernel/CVE-2016-6187.yml

```diff
@@ -1,479 +1,497 @@
+    was wrong. Otherwise, leave it blank.
 bounty:
-  amt:
-  announced:
-  url:
+    amt:
+    announced:
+    url:
 reviews: []
 bugs_instructions: |
-  What bugs are involved in this vulnerability?
+    What bugs are involved in this vulnerability?
 
-  Please list bug IDs to https://bugzilla.kernel.org/
+    Please list bug IDs to https://bugzilla.kernel.org/
 
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
+    Bug ID's can appear in several places:
+      * Mentioned in commit messages
+      * Mentioned in mailing list discussions
+      * References from NVD entry
+      * Various other places
 bugs: []
 fixes_instructions: |
-  Please put the commit hash in "commit" below.
+    Please put the commit hash in "commit" below.
 
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
+    This must be a git commit hash from the systemd source repo, a  40-character
+    hexademical string/
 
-  Place any notes you would like to make in the notes field.
+    Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
-- commit: 30a46a4647fd1df9cf52e43bf467f0d9265096ca
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    - commit: 30a46a4647fd1df9cf52e43bf467f0d9265096ca
+      note: "Manually confirmed"
 vcc_instructions: |
-  The vulnerability-contributing commits.
+    The vulnerability-contributing commits.
 
-  These are found by our tools by traversing the Git Blame history, where we
-  determine which commit(s) introduced the functionality.
+    These are found by our tools by traversing the Git Blame history, where we
+    determine which commit(s) introduced the functionality.
 
-  Look up these VCC commits and verify that they are not simple refactorings,
-  and that they are, in fact introducing the vulnerability into the system.
-  Often, introducing the file or function is where the VCC is, but VCCs can be
-  anything.
+    Look up these VCC commits and verify that they are not simple refactorings,
+    and that they are, in fact introducing the vulnerability into the system.
+    Often, introducing the file or function is where the VCC is, but VCCs can be
+    anything.
 
-  Place any notes you would like to make in the notes field.
+    Place any notes you would like to make in the notes field.
 vccs:
-- commit: b5e95b48685e3481139a5634d14d630d12c7d5ce
-  note: Discovered automatically by archeogit.
-- commit: 3eea57c26e49a5add4c053a031cc2a1977b7c48e
-  note: Discovered automatically by archeogit.
+    - commit: b5e95b48685e3481139a5634d14d630d12c7d5ce
+      note: Discovered automatically by archeogit.
```

Comment:
I would recommend adding some notes to the commits describing briefly what they break to be considered a vcc. With so many contributing commits, I suspect a couple may not actually be too relevant.